### PR TITLE
MINOR: Add William Hyun to committers.md

### PR DESCRIPTION
**Pull request by williamhyun**
### What changes were proposed in this pull request?
This PR aims to add `William Hyun` to the committers.md file. 

### Why are the changes needed?
N/A

### How was this patch tested?
N/A

### CODE REVIEW

File: site/develop/committers.md
Comment (pgaref) [inline site/develop/committers.md:49]: Order is alphabetical so it should probably be after Wei but feel free to merge after that

Comment (dongjoon-hyun) [reply site/develop/committers.md:49]: The order is alphabetical in ID, isn't it?

**Review by dongjoon-hyun — APPROVED** [review]
Ya, you should merge by yourself, @williamhyun .
Please re-check the order according to @pgaref 's comment (names or ids) and merge this.

File: site/develop/committers.md
Comment (dongjoon-hyun) [reply site/develop/committers.md:49]: It seems that ID-order.
```
Gang Wu                 | gangwu       | PMC
Alan Gates              | gates        | PMC
```

Comment (dongjoon-hyun) [reply site/develop/committers.md:49]: ID-order looks dominant. We had better fix the sorting in a separate PR, @williamhyun and @pgaref .
```
Matthew McCline         | mmccline     | Committer
Sandeep More            | moresandeep  | Committer
Naveen Gangam           | ngangam      | Committer
```

**Comment by williamhyun** [comment]
@pgaref @dongjoon-hyun Thank you for your review,
I thought the order was in alphabetical order with the IDs. 
I will merge this and create a separate PR addressing this issue.

**Comment by williamhyun** [comment]
It seems that I don't have the committer permission yet. I cannot see `merge` button in GitHub and I cannot push via CLI.
```
william@Williams-MacBook-Pro orc % git push apache HEAD:master
ERROR: Permission to apache/orc.git denied to williamhyun.
fatal: Could not read from remote repository.
```

**Comment by dongjoon-hyun** [comment]
Oh.. Got it. 

Hi, @omalley . It seems that we need the power of PMC Chair. Could you give the committer permission to @williamhyun ?

**Comment by omalley** [comment]
William is already listed as an ORC committer: https://people.apache.org/phonebook.html?unix=orc
He probably needs to update https://id.apache.org/ with his github id.

**Comment by williamhyun** [comment]
Hello @omalley , I already updated id.apache.org with my GitHub id. 
Screen Shot 2021-04-16 at 6 16 07 PM

But I cannot see the merge button. 
Screen Shot 2021-04-16 at 6 18 05 PM

**Comment by williamhyun** [comment]
I switched the lines to match the ID-alphabetical order.
```
- William Hyun            | william      | Committer
  Wei Zheng               | weiz         | Committer
+ William Hyun            | william      | Committer
```

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM.

Hi, @williamhyun . Could you try to push `gitbox.apache.org` instead of `github.com` ?

**Comment by williamhyun** [comment]
I will try gitbox.

**Comment by williamhyun** [comment]
It seems that gitbox works! Thank you all for your assistance.

**Comment by dongjoon-hyun** [comment]
Congratulations for your first merging, @williamhyun !

**Pull request merged**
